### DISCUSSION TITLE
Battlefield 2042 can not launch with Windhawk open and Disable grouping on the taskbar Enable

### DISCUSSION THREAD

**Discussion by Psyc0tik** [ramensoftware/windhawk#164] [General]
Battlefield 2042 can not launch on Steam with Windhawk open and Disable grouping on the taskbar Enable
Hard to find that is the bug
Solution to boot the game is to close Windhawk

**Comment by m417z** [comment]
That's probably related to the game's anti-cheat mechanism. You can exclude it in Windhawk: configure it [in the advanced settings](https://i.imgur.com/ktPDUNn.png) and add `Bf2042.exe` (or the relevant exe name or path) to the process exclusion list.

**Reply by gempir** [reply]
Question, why does Windhawk by default want to hook into all processes? I assume each plugin could provide that list.

Wouldn't it make more sense to keep an allowlist? 

Since a lot of gamers will probably have issues with Anti-Cheat and will find it hard to figure this out.

**Reply by m417z** [reply]
It was brought up several times, please see my answer here: https://github.com/ramensoftware/windhawk/discussions/21.
Also see this wiki page:
https://github.com/ramensoftware/windhawk/wiki/Injection-targets-and-critical-system-processes#why-does-windhawk-inject-code-into-most-processes

It's planned to have paths of well-known game and conflicting programs excluded by default: https://github.com/ramensoftware/windhawk/issues/402.

**Comment by Cris--** [comment]
Path or filenames with spaces needs "" ?

sry 4 hijacking here..

**Reply by m417z** [reply]
No, the paths must be without quotes.
